Networking/Sources/Networking/Requests/ACIListRequests/CategoriesListRequest.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+// MARK: - CategoriesListResponse
+public struct CategoriesListResponse: Decodable {
+    let meals: [CategoryMeal]
+}
+
+// MARK: - Meal
+struct CategoryMeal: Decodable {
```

Comment:
What is a category meal? It doesn't reflect what it means. 

A good SWE should write an API that is readable from his SELF Future and for the team.